cirq/ops/common_gates.py

```diff
@@ -889,70 +911,101 @@ def Rz(rads: float) -> ZPowGate:
     return ZPowGate(exponent=rads / np.pi, global_shift=-0.5)
 
 
-# The Pauli X gate.
-#   [[0, 1],
-#    [1, 0]]
 X = XPowGate()
+"""The Pauli X gate.
+
+Matrix:
+
+    [[0, 1],
+     [1, 0]]
+"""
 
 
-# The Pauli Y gate.
-#   [[0, -i],
-#    [i, 0]]
+#: The Pauli Y gate.
```

Comment:
Is this `#: ` another way to set a docstring? I haven't seen that before.

Reply:
I'm leaving this in for now, because it will be iterated on more in later PRs.